frontend/components/TitleBar.vue

```diff
@@ -1,19 +1,41 @@
 <script setup lang="ts">
 import { onMounted } from 'vue'
 
+const router = useRouter()
+
 interface Props {
   title: string
 }
 const props = withDefaults(defineProps<Props>(), {})
+
+const back = () => { router.go(-1) }
+
 onMounted(() => {
   document.title = 'RMI | PACTA | ' + props.title
 })
 </script>
 
 <template>
-  <h1 class="inline-block text-primary title-bar">
-    {{ title }}
-  </h1>
+  <div class="relative flex flex-column gap-2 align-items-start">
+    <PVButton
+      icon="pi pi-angle-left"
+      class="p-button-secondary p-button-text md:hidden p-0 text-600 pr-2"
+      style="margin-left: -.25rem"
+      label="Back"
+      @click="back"
+    />
```

Comment:
I can see the value in a 'breadcrumb'-style system that has a list of your current 'depth' into some entity, but less so for just replicating the browser's own "go back" functionality. I'd remove this unless there's some core value prop you have in mind